test/tests.py

```diff
@@ -200,3 +221,37 @@ def test_0006_invalidIncorrectUri(self):
         incorrectUri = "/_cluster/incorrectUri"
         response = requests.get(f'{self.proxy_endpoint}{incorrectUri}', auth=self.auth, verify=False)
         self.assertEqual(response.status_code, HTTPStatus.METHOD_NOT_ALLOWED)
+
+    def test_0007_OSB(self):
+        cmd = "docker ps | grep 'migrations/migration_console:latest' | awk '{print $NF}'"
```

Comment:
I recommend using `docker ps --format="{{.ID}}" --filter name=migration-console` and avoiding the pipes